Makefile

```diff
@@ -16,4 +16,10 @@ docker-run:
 	layer5/meshery-istio
 
 run:
-	DEBUG=true GOPROXY=direct GOSUMDB=off go run main.go
\ No newline at end of file
+	DEBUG=true GOPROXY=direct GOSUMDB=off go run main.go
+
+test: 
+	brew install nektos/tap/act
```

Comment:
What about users on windows or ubuntu ?

Reply:
@kushthedude I am using Ubuntu, and i is working as expected. But for windows, we can use scoop. This is my approach to this issue. if it is the right approach then I will make it for windows also. for Linux and mac, this will work

Reply:
I think ok to start with support for a single OS and see if we want to invest more.